src/utils/searchResults.js

```diff
@@ -0,0 +1,19 @@
+/* @flow */
```

Comment:
Not only that, but I guess we are expecting a utility function to not be aware of React components.

This might be more appropriate as a new component.